pkg/controller/statics/statics_controller_test.go

```diff
@@ -166,19 +168,30 @@ func TestReconcileCRDVariants(t *testing.T) {
 
 	// Restores the state where a) our statics exist, b) the CRD is green.
 	reset := func() {
-		r.client.Delete(ctx, crd)
-		r.client.Create(ctx, &apiextensions.CustomResourceDefinition{
+		err = r.client.Delete(ctx, crd)
+		if errors.IsNotFound(err) {
+		} else if err != nil {
+			panic(err)
+		}
```

Comment:
style nit, this would more commonly be spelled:
```suggestion
		if err = r.client.Delete(ctx, crd); err != nil && !errors.IsNotFound(err) {
			panic(err)
		}
```
But also, in the error paths below you're using `t.Fatal(err)`, so you might as well do the same here
```suggestion
		if err = r.client.Delete(ctx, crd); err != nil && !errors.IsNotFound(err) {
			t.Fatal(err)
		}
```